workers/data_refinery_workers/processors/illumina.py

```diff
@@ -216,50 +320,55 @@ def _create_result_objects(job_context: Dict) -> Dict:
     result.pipeline = "Illumina SCAN"
     result.save()
 
-    # Create a ComputedFile for the sample,
-    # sync it S3 and save it.
-    try:
+    # Split the result into smashable subfiles
+    big_tsv = job_context["output_file_path"]
+    data = pd.read_csv(big_tsv, sep='\t', header=0, index_col=0)
+    individual_files = []
+    frames = np.split(data, len(data.columns), axis=1)
+    for frame in frames:
+        frame_path = job_context["output_file_path"] + '.' + frame.columns.values[0].replace('&', '') + '.tsv'
+        frame.to_csv(frame_path, sep='\t', encoding='utf-8')
+
+        # This needs to be the same as the ones in the job context!
+        sample = Sample.objects.get(accession_code=frame.columns.values[0])
```

Comment:
When you build the path above you do a `frame.columns.values[0].replace('&', '')`. Will the accession codes in the database contain these ampersands?